hack/release-scripts/get-latest-sidecar-images

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ---
+# This script generates a copy of image-digests-template.yaml with the latest tags and associated manifest digests
+# for each sidecar image at OUTPUT_FILEPATH.
+
+# --- Environment Variables
+export ROOT_DIRECTORY IMAGE_DIGESTS_TEMPLATE_FILEPATH OUTPUT_FILEPATH
+ROOT_DIRECTORY=${ROOT_DIRECTORY:=$(git rev-parse --show-toplevel)}
+IMAGE_DIGESTS_TEMPLATE_FILEPATH=${IMAGE_DIGESTS_TEMPLATE_FILEPATH:="$ROOT_DIRECTORY/hack/release-scripts/image-digests-template.yaml"}
+OUTPUT_FILEPATH=${OUTPUT_FILEPATH:="$ROOT_DIRECTORY/hack/release-scripts/image-digests.yaml"}
+
+tmp_filename="tmp_$RANDOM.txt"
```

Comment:
Continuing the discussion from the previous version of this PR: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1791#discussion_r1365937071

I'm not sure what `mktemp` incompatibility you're referring to, can you specify. The SO post you link refers to `mktemp` creating a directory, not a file, and also appears to be fixed in recent versions of MacOS.

(also applicable in other file)

Reply:
To be more explicit, I attempted using `mktemp` in this script after your original comment, but the script failed to run on MacOs. See this separate StackOverflow post for more information on the topic [macos - Why is mktemp on OS X broken with a command that worked on Linux? - Stack Overflow](https://stackoverflow.com/questions/31396985/why-is-mktemp-on-os-x-broken-with-a-command-that-worked-on-linux). 

Either way, thank you for bringing up `mktemp`, but the script's current implementation of creating a temporary file is more portable and satisfies the script's needs.